yesod-bin/main.hs

```diff
@@ -30,12 +30,14 @@ data Command = Init [String]
              | Build { buildExtraArgs   :: [String] }
              | Touch
              | Devel { develSuccessHook :: Maybe String
-                     , develExtraArgs    :: [String]
+                     , develExtraArgs   :: [String]
                      , develPort        :: Int
                      , develTlsPort     :: Int
                      , proxyTimeout     :: Int
                      , noReverseProxy   :: Bool
                      , develHost        :: Maybe String
+                     , certPath         :: Maybe FilePath
```

Comment:
If you combine these into a single field `cert :: Maybe (FilePath, FilePath)`, the opt parser can enforce that either both arguments or neither argument are provided, which will perhaps reduce user error

Reply:
Took me a while to figure this out as I'm not very familiar with the parser library. I couldn't find a function in the library docs which would require both or neither value but I think what I've just implemented is a reasonable solution.